default.yml

```diff
@@ -41,6 +41,14 @@ Metrics/BlockLength:
     # Specs deliberately will almost always break this cop (It should be added to rubocop-rspec defaults ideally)
     - spec/**/*
```

Comment:
I've fixed this (Kind of with help), in the gem proper: https://github.com/rubocop/rubocop-rspec/pull/1256
Released in 2.11 - https://github.com/rubocop/rubocop-rspec/blob/master/CHANGELOG.md#2110-2022-05-18